content/en/docs/concepts/configuration/configmap.md

```diff
@@ -61,6 +61,10 @@ You can write a Pod `spec` that refers to a ConfigMap and configures the contain
 in that Pod based on the data in the ConfigMap. The Pod and the ConfigMap must be in
 the same {{< glossary_tooltip text="namespace" term_id="namespace" >}}.
 
+{{< note >}}
+{{< glossary_tooltip text="Static Pods" term_id="static-pod" >}} do not support ConfigMap.
```

Comment:
This is a little too glib.  Note the list below of four ways that a pod can use a ConfigMap. For the fourth one, "Write code to run inside the Pod that uses the Kubernetes API to read a ConfigMap", the story is bigger and more complicated.  A static pod can not use a ServiceAccount, and so can not use the Kubernetes API at all with credentials from a ServiceAccount.  OTOH, a static pod that gets credentials by some other means (for example, how kubeadm gives credentials to the static pods it defines for the controller manager and scheduler) _can_ use the Kubernetes API --- including ConfigMaps and Secrets.

Reply:
I'm a little confused. So static pod may or may not support configmap/secret/serviceaccount depending on whether it get credentials? In such case, the existing statement I added is wrong. And it's too much detail to explain how static pod works in this page. We could remove the statement in this page, and explain configmap/secret/serviceaccount for static pod in its own page (maybe in https://kubernetes.io/docs/concepts/workloads/pods/#static-pods). It seems there's more discussion for this topic. I will wait for more information before update it.

Reply:
The problem is that simply speaking of "support ConfigMap" is too vague.  The rule is really very simple and I think we can draft a concise statement of it.  For the note at this point in the doc, we could say something like the following.

> The Spec of a {{< glossary_tooltip text="Static Pods" term_id="static-pod" >}} can not refer to a {{< glossary_tooltip text="ConfigMap" term_id="configmap" >}} or any other API object.